### PR TITLE
West Midlands Class 3 Week 1 Anna Clayton-Wright

### PR DESCRIPTION
**Pull request by Aclayonwright**
updated wrong answers

# Your Details

Anna Clayton-Wright
Birmingham
Anna CW

# Homework Details
Module 1
- Module:
- Week:

-----
[View rendered homework.md](https://github.com/Aclayonwright/GitHomeworkFixErrors/blob/main/homework.md)

### CODE REVIEW

File: homework.md [hunk 1]
```diff
@@ -44,61 +44,61 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
 
-A place for us to store and share our code.
+Provider of interet hsing for software development
 
 ## Question 15
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+Save you code to Github
```
Comment (tenzyns) [inline homework.md:95]: It means, to save the changes made to GitHub/desktop.

**Review by tenzyns — COMMENTED** [review]
I have completed the review.

File: homework.md [hunk 2]
```diff
@@ -4,38 +4,38 @@
...
-Making a website go faster
+Presentation and style
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
 ```css
-#h1 {
+#h1 {
```
Comment (justsealan) [inline homework.md:38]: #h1 only selects the elements with id = "h1". To select all the h1 elements you can just write plain h1.

File: homework.md [hunk 3]
```diff
@@ -44,61 +44,61 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
 
-When people want to show off code to each other they can put it on Github
+collaboration,  make changes to other people code and merge the code, platform for viewing repositories.
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You take a repository from one account to another.
```
[the remark this answers is not in the document]
Comment (justsealan) [reply homework.md:59]: I think both are correct.

File: homework.md [hunk 4]
```diff
@@ -44,61 +44,61 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
 
 ## Question 10 
 
 What happens when you clone a repostory?
 
-It send it to a friend
+it goes onto your system.
```
[the remark this answers is not in the document]
Comment (justsealan) [reply homework.md:65]: Same thing 👍

File: homework.md [hunk 5]
```diff
@@ -44,61 +44,61 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+telling others about changes you have mde to a repo.
```
[the remark this answers is not in the document]
Comment (justsealan) [reply homework.md:71]: You both are correct 👍

File: homework.md [hunk 6]
```diff
@@ -44,61 +44,61 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
 
-You send the changes of your local repository to Github
+makes the changes you have made accessible to others.
 
 ## Question 14
 
 What is Github?
 
-A place for us to store and share our code.
+Provider of interet hsing for software development
```
[the remark this answers is not in the document]
Comment (justsealan) [reply homework.md:89]: It's a hosting service and it also allows us to control the old version of the codes and share.

File: homework.md (hunk 1 above)
Comment (justsealan) [reply homework.md:95]: Yes saves the code to Github 👍

**Review by justsealan — APPROVED** [review]
Great work! Well Done!

**Pull request closed without merging**